---
layout: page
title: Software
permalink: /Tools/Software/
menu: tools
published: true
---

### XYZ

details

* [xyz](https://github.com/parisodlab/xyz)


### EasyXpress

Our lab has developed an R package to manage, process, and plot high-throughput imaging data obtained using the Molecular Devices ImageXpress platform. 

* [easyXpress](https://github.com/AndersenLab/easyXpress)

### EasyFulcrum

For labs interested in field sampling wild strains, we built an R package to read-in, process, analyze, and plot sampling data collected using Fulcrum (apps are below). A procotol for nematode sampling is [here](https://www.jove.com/t/63486/a-highly-scalable-approach-to-perform-ecological-surveys-selfing).

* [easyFulcrum](https://github.com/AndersenLab/easyfulcrum)


### [VCF-kit](http://www.github.com/andersenlab/VCF-kit) - [Documentation](https://github.com/AndersenLab/VCF-kit)

---

<a href='http://www.github.com/andersenlab/VCF-kit'><img src='/assets/img/vcfkit.png' style='float:left; width: 150px; margin-right: 20px;' /></a>

VCF-kit is a command-line based collection of utilities for performing analysis on Variant Call Format (VCF) files. A summary of the commands is provided below. See documentation for details on installation and usage.

<div style='clear:both;'></div>
<br />

<table class='table table-hover'><thead>
<tr>
<th align="left">Command</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td align="left"><strong>calc</strong></td>
<td>Obtain  frequency/count of  genotypes and alleles.</td>
</tr>
<tr>
<td align="left"><strong>call</strong></td>
<td>Compare variants  identified  from  sequences obtained  through alternative methods against a VCF.</td>
</tr>
<tr>
<td align="left"><strong>filter</strong></td>
<td>Filter  variants  with  a minimum or  maximum number  of  REF,  HET,  ALT,  or  missing calls.</td>
</tr>
<tr>
<td align="left"><strong>geno</strong></td>
<td>Various operations  at  the genotype  level.</td>
</tr>
<tr>
<td align="left"><strong>genome</strong></td>
<td>Reference genome  processing  and management.</td>
</tr>
<tr>
<td align="left"><strong>hmm</strong></td>
<td>Hidden-markov model for use in  imputing  genotypes from  parental  genotypes in  linkage studies.</td>
</tr>
<tr>
<td align="left"><strong>phylo</strong></td>
<td>Generate  dendrograms from  a VCF.</td>
</tr>
<tr>
<td align="left"><strong>primer</strong></td>
<td>Generate  primers for variant validation.</td>
</tr>
<tr>
<td align="left"><strong>rename</strong></td>
<td>Add a prefix, suffix, or  substitute  a string  in  sample  names.</td>
</tr>
<tr>
<td align="left"><strong>tajima</strong></td>
<td>Calculate Tajimaâ€™s  D.</td>
</tr>
<tr>
<td align="left"><strong>vcf2tsv</strong></td>
<td>Convert a VCF to  TSV.</td>
</tr>
</tbody></table>

<div style='clear:both;'></div>
<br />
